apps/src/applab/PlaySpaceHeader.jsx

```diff
@@ -35,12 +35,22 @@ var PlaySpaceHeader = React.createClass({
 
   render: function () {
     var leftSide, rightSide;
+    var showDataModeButton = Applab.useFirebase;
```

Comment:
Rather than coupling this component with Applab, I think we should be putting useFirebase into our redux store (presumably as a pageConstant)